<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QTableWidget" name="tableStudent">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>外观设置</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QCheckBox" name="checkHHeader">
                 <property name="text">
                  <string>行表头</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkVHeader">
                 <property name="text">
                  <string>列表头</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkAltemate">
                 <property name="text">
                  <string>交替背景色</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkEditCell">
                 <property name="text">
                  <string>单元格可编辑</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnStyleSheet">
              <property name="text">
               <string>设置样式表</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>选择设置</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="radioCell">
              <property name="text">
               <string>选择单元格</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioRow">
              <property name="text">
               <string>选择行</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>数据操作</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>姓名</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineAge"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineName"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineGender"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineProvince"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>性别</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>年龄</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>籍贯</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="btnAdd">
             <property name="text">
              <string>添加一行</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnInsert">
             <property name="text">
              <string>插入一行</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDel">
             <property name="text">
              <string>删除当前行</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnUpdate">
             <property name="text">
              <string>修改当前行</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
